### PR TITLE
fix bug with contact dialog and added trades' list

### PR DESCRIPTION
**Pull request by letrylesha**

### CODE REVIEW

File: packages/raccoon-city-ui/src/components/Main/Contacts/Contact/Contact.tsx [hunk 1]
```diff
@@ -30,6 +30,14 @@ export const Contact = ({open, handleClose, uuid}) => {
         return null;
     }
 
+    // TODO think about it
+    if (!open) {
+        return null;
+    }
+    if (!uuid) {
+        return null;
+    }
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/Contacts/Contact/Contact.tsx:39]: !open || !uuid then :)

File: packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    createStyles({
+        root: {
+            width: '100%'
+        },
+        heading: {
+            fontSize: theme.typography.pxToRem(15),
+            fontWeight: theme.typography.fontWeightRegular
+        }
+    })
+);
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx:24]: remove :) Use Styled Components instead

File: packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+        return <div>Loading</div>;
+    }
+
+    return (
+        <div className={classes.root}>
+            {trades.map((item) => (
+                <Accordion key={item.id}>
+                    <AccordionSummary aria-controls="panel1a-content" id={item.id}>
+                        <span>
+                            {`№: ${item.tradeNumber} Cтатус: ${item.leadStatus} Квартира: ${item.flat.flatNumber}`}
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx:39]: Add ЖК - Дом - Квартира

File: packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+                                        <Typography variant="body2" component="p">
+                                            {item.state}
+                                        </Typography>
+                                    </TableCell>
+                                </TableRow>
+
+                                <TableRow key={item.key}>
+                                    <TableCell component="th" scope="row">
+                                        <Typography variant="body2" component="p">
+                                            budget
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx:61]: Use Russian please

File: packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactForm.tsx [hunk 5]
```diff
@@ -61,16 +62,13 @@ export function ContactForm({onClose, contact}) {
     const {data, loading, error} = useQuery(GET_USERS);
     const {data: dropdowns, loading: dropdownsLoading} = useQuery(GET_CONTACT_DROPDOWNS);
 
-    const {data: trades} = useQuery(APPROPRIATE_TRADES, {
+    const {data: trades, loading: isLoadingTrade} = useQuery(APPROPRIATE_TRADES, {
         fetchPolicy: 'cache-and-network',
         variables: {
             contactId: contact.id
         }
     });
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactForm.tsx:71]: I think it is better to query appropriate trades directly in ContactTrades but not in a parent component :)

File: packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+import {GET_TRADE_DROPDOWNS} from '../../../../graphql/queries/tradeQuery';
+import {format} from 'date-fns';
+import {getConstant, getClientInterests} from './ContactTradesUtils';
+import {RootContainer, TradeTitleContainer, Bold} from './ContactForm.styled';
+
+interface Props {
+    trades: any;
+}
+
+const ContactTrades: React.FC<Props> = ({trades}) => {
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx:18]: Don't type React.FC 
better to have (props: Props (use ComponentNameProps))

File: packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+                                <Bold>Статус :</Bold>&nbsp;{getConstant(dropdowns, item.leadStatus, 'leadStatuses')}
+                            </p>
+                            <p>
+                                <Bold>ЖК :</Bold>
+                                {item.flat.apartmentComplex}
+                                <Bold>Дом :</Bold>
+                                {item.flat.house}
+                                <Bold>Квартира :</Bold>
+                                {item.flat.flatNumber}
+                            </p>
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx:42]: Could you please in subsequent PRs attach screenshots?

File: packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactForm.tsx (hunk 5 above)
Comment (letrylesha) [reply packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactForm.tsx:71]: make sense

File: packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx (hunk 7 above)
Comment (letrylesha) [reply packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx:42]: yep

**Pull request merged**